src/workerd/api/urlpattern-standard.h

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include "ada.h"
+
+#include <workerd/jsg/jsg.h>
+
+#include <kj/array.h>
+#include <kj/one-of.h>
+#include <kj/string.h>
+
+#include <optional>
+#include <string>
+#include <string_view>
+#include <utility>  // For std::move
```

Comment:
kj::mv expressly exists as a shorter replacement for std::move that doesn't require pulling in <utility>: https://github.com/capnproto/capnproto/blob/v1.1.0/c%2B%2B/src/kj/common.h#L692

Reply:
TIL. Thanks!